proto/tendermint/types/evidence.proto

```diff
@@ -25,7 +15,24 @@ message Evidence {
   }
 }
 
-// EvidenceData contains any evidence of malicious wrong-doing by validators
-message EvidenceData {
+// DuplicateVoteEvidence contains evidence of a validator signed two conflicting votes.
+message DuplicateVoteEvidence {
+  tendermint.types.Vote       vote_a = 1;
+  tendermint.types.Vote       vote_b = 2;
+  int64                       total_voting_power = 3;
```

Comment:
nit: spec updates are needed

Reply:
not a nit! this is actually important and a good thing to point out :)